fdb-record-layer-core/src/main/proto/record_query_plan.proto

```diff
@@ -1502,6 +1489,14 @@ message PRecordQueryInsertPlan {
   optional PRecordQueryAbstractDataModificationPlan super = 1;
 }
 
+//
+// PTempTableInsertPlan
+//
+message PTempTableInsertPlan {
+  optional PRecordQueryAbstractDataModificationPlan super = 1;
+  optional PValue temp_table = 2;
```

Comment:
should be the value.

Reply:
renamed to `temp_table_reference_value`.